maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/AbstractSurefireMojo.java

```diff
@@ -847,12 +847,33 @@ private DefaultScanResult scanDependencies()
         {
             try
             {
+                DefaultScanResult scanResult = new DefaultScanResult( Collections.EMPTY_LIST );
+                
+                List<String> dependenciesToScanList = new ArrayList( Arrays.asList( getDependenciesToScan() ) );
```

Comment:
`dependenciesToScanList` already contains plural. Pls rename `dependenciesToScanList` to `dependenciesToScan`.